### PR TITLE
[Office add-ins march 2022] Remove consuming graph services demo

### PR DESCRIPTION
**Pull request by davidchesnut**
## Category

- [x] Content fix
- [ ] New article

## Related issues

None

## Contents of the Pull Request

Update blog post since demo was removed.

### CODE REVIEW

File: content/office-add-ins-community-call/office-add-ins-community-call-march-9-2022/index.md [hunk 1]
```diff
@@ -12,21 +12,20 @@ type: "regular"
...
 *   **Excel API 1.15 preview** – Sirui Sun, Software Engineer (Microsoft) – [21:08](https://youtu.be/95qxmmqWUAk&t=1268)
 *   **Q&A** – [25:45](https://youtu.be/95qxmmqWUAk&t=1545)
 
+> Last updated: July 7. The presentation **Consuming Graph Services using SSO and CORS from event–based Outlook add–ins** was removed from this call. Upon further research, it came to our attention that this approach has a potential security risk. Once the Microsoft Graph token is obtained by the middle-tier, it sends the token to the add-in's client code running in Outlook. The Microsoft Graph access token created by the Microsoft identity platform as part of the on-behalf-of flow is only intended to be used by the middle tier. Find more details about this security risk at [Middle-tier access token request](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow#middle-tier-access-token-request).
```
Comment (AlexJerabek) [inline content/office-add-ins-community-call/office-add-ins-community-call-march-9-2022/index.md:26]: Consistent hyphens.
```suggestion
> Last updated: July 7. The presentation **Consuming Graph Services using SSO and CORS from event–based Outlook add–ins** was removed from this call. Upon further research, it came to our attention that this approach has a potential security risk. Once the Microsoft Graph token is obtained by the middle-tier, it sends the token to the add-in's client code running in Outlook. The Microsoft Graph access token created by the Microsoft identity platform as part of the on-behalf-of flow is only intended to be used by the middle-tier. Find more details about this security risk at [Middle-tier access token request](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow#middle-tier-access-token-request).
```

**Review by AlexJerabek — APPROVED** [review]
Approving with a slight grammar suggestion.

**Comment by davidchesnut** [comment]
@LuiseFreese do you know how to fix the Markdown Lint network errors in this PR? It looks like some links to valid URLs fail for some reason in the link check.

**Comment by LuiseFreese** [comment]
Hey there @davidchesnut - Don't worry about the Linkchecker. As we use a static site generator (Hugo), the entire site gets built every time we push/merge. That also means that all links of all articles get a check. 

Sometimes, perfectly fine working links are flagged as network error, sometimes, we see that an SSL certificate has expired. I will take care of that :-)

**Pull request merged**